README.md

```diff
@@ -18,41 +18,20 @@ body) of requests. It also does not aim to be the fastest, or most secure.
 
 Java (version 16 or later). No third-party libraries required for execution.
 
-Dependencies for build / test (such as JUnit) are managed using Gradle. All dependencies should be current, stable
-versions.
-
-## Building
-
-### With Gradle
-
-ProxyAuth can be built using Gradle, and includes the
-[Gradle Wrapper](https://docs.gradle.org/7.2/userguide/gradle_wrapper.html). Note that Gradle has different
-requirements, such as not currently supporting Java 17 or later.
-
-- On Windows:
-  `gradlew build`
-- On Linux / Unix:
-  `./gradlew build`
-
-Gradle will create the jar file in `build\libs` (e.g. `build\libs\ProxyAuth-0.1.0.jar`)
-
-### Without Gradle
-
-It does not rely on any third-party libraries, so can be built using any Java IDE, or even javac.
-
-`C:\example\ProxyAuth\src\main\java>javac -d ..\bin proxyauth\*.java`
-
-## Installation
+## Running
 
 TODO - depends on https://github.com/Zeckie/ProxyAuth/issues/6 (configuration)
 
-ProxyAuth does not have a GUI, and should be run from a console / shell such as `cmd.exe`.
-`java -jar ProxyAuth-0.1.0.jar`
+ProxyAuth does not have a GUI, and should be run from a console / shell such as `bash` / `powershell`.
```

Comment:
powershell - is that a joke?

Reply:
I think cmd isn't being developed anymore, (https://www.theregister.com/2020/05/27/windows_cmd_maintenance_mode/), it's more like a legacy tool.

Reply:
> Cmd is one of the most frequently run executables on Windows ...

> The Windows Cmd / Command-Line shell is NOT being removed from Windows in the near or distant future!

https://devblogs.microsoft.com/commandline/rumors-of-cmds-death-have-been-greatly-exaggerated/

Also, looks like Gradle Wrapper still uses cmd.exe (`gradlew.bat` not `gradlew.ps1`)

Reply:
I've updated readme to mention powershell